packages/realm/src/Realm.ts

```diff
@@ -478,6 +632,15 @@ export class Realm {
     }
   }
 
+  /**
+   * Searches for a Realm object by its primary key.
+   * @param type - The type of Realm object to search for.
+   * @param primaryKey - The primary key value of the object to search for.
+   * @throws If type passed into this method is invalid or if the object type did
+   *   not have a `primaryKey` specified in its {@link ObjectSchema}.
```

Comment:
Does this link resolve correctly?

Reply:
links seem to work weirdly... they don't really link to meaningful places (they do link to code) so I saw thinking of removing them.